egograph/pipelines/domain/errors.py

```
@@ -23,3 +23,7 @@ class StepRunNotFoundError(PipelinesError):
 
 class WorkflowLockUnavailableError(PipelinesError):
     """workflow lock を取得できない。"""
+
+
+class AuthenticationError(PipelinesError):
+    """認証エラー（トークン失効・無効なクレデンシャル等）。リトライ対象外。"""
```

Ruff (0.15.15)

[warning] Docstring contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF002)